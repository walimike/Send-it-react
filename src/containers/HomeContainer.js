import React, { Component } from 'react';


import CreateParcelContainer from './CreateParcel';
import RetrieveParcels from './RetrieveParcels';

export default class HomeContainer extends Component {

  logout=()=>{
    sessionStorage.clear();
    /* istanbul ignore next */
    window.location.href="/"
  }

  render() {
    return (
        <div>
            <nav>
          <div className="nav-wrapper red">
            <a href="#!" className="brand-logo left">Send it</a>
            <ul className="right hide-on-med-and-down">
              <li className="active"><a  onClick={this.logout}>Logout</a></li>
            </ul>
          </div>
        </nav>
                <div className="container">
        <div className="row">
        <div className="col s6">
            <CreateParcelContainer/>
        </div>
        <div className="col s6">
            <RetrieveParcels/>
        </div>
        </div>
        
        
      </div>
        </div>
    )
  }
}
