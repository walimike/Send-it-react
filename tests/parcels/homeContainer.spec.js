import React from 'react';
import { shallow } from 'enzyme';

import HomeContainer from '../../src/containers/HomeContainer';

describe('Home Container', () => {
  const props = {
    redirectUser: jest.fn()
  }
  it('should render without crashing', () => {
    const wrapper = shallow(<HomeContainer />);
    expect(wrapper).toMatchSnapshot();
  });

  it('should call logout', () => {
    const wrapper = shallow(<HomeContainer />);
    wrapper.instance().logout()
  })

});