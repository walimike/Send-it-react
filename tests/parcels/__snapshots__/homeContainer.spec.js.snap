// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home Container should render without crashing 1`] = `ShallowWrapper {}`;
